content/metadata.yaml

```diff
@@ -87,3 +177,19 @@ author_info:
       - Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison
       - Morgridge Institute for Research
     correspondence: true
+    roles:
+      - Conceptualization: true
+      - Data curation: true
+      - Formal analysis: true
```

Comment:
You can flip this to `false`.  No one else has it set as `true`.  Everything else in my roles looks good.

The other authors' contributions look reasonable as well.